### PR TITLE
Add label support

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Add support for labels.

For more context/info see: https://github.com/home-assistant/core/pull/69996

Needs: <https://github.com/home-assistant/home-assistant-js-websocket/pull/383>

CleanShot 2023-11-04 at 17 47 29@2x

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by johntdyer** [comment]
@frenck  any plans to pick this PR up after summer ?   Not that I am qualified but if there is any scut work I can do to help facilitate it I would be more then happy to volunteer

**Comment by johntdyer** [comment]
image

File: src/data/label_registry.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+export interface LabelEntityLookup {
+  [labelId: string]: EntityRegistryEntry[];
+}
+
+export interface LabelDeviceLookup {
+  [labelId: string]: DeviceRegistryEntry[];
+}
+
+export interface LabelRegistryEntryMutableParams {
```
Comment (bramkragten) [inline src/data/label_registry.ts:25]: .```suggestion
export type LabelRegistryEntryMutableParams = Omit<"label_id", LabelRegistryEntry>
```

**Comment by bendikrb** [comment]
🤔

**Pull request closed without merging**